appeals/e2e/cypress/page_objects/basePage.js

```diff
@@ -66,6 +67,7 @@ export class Page {
 			cy.get(this.selectors.accordion).contains('span', text, { matchCase: false }),
 		accordionButton: (text) =>
 			cy.get(this.selectors.accordionButton).contains('span', text, { matchCase: false }),
+		additionalDocumentsAdd: () => cy.get(this.selectors.summaryCardActions).children(),
```

Comment:
what does.children() do?

Reply:
it's a cypress method to grab the children of a parent element.